src/Storages/System/StorageSystemZooKeeperInfo.cpp

```diff
@@ -0,0 +1,392 @@
+            if (!attribute_value.empty())
+                index = stoi(attribute_value);
+        }
+
+        res_columns[3]->insert(index);
+
+
+        /// ruok command
+        String ruok_output = sendFourLetterCommand(host, port, "ruok");
+        if (ruok_output == "imok")
+            res_columns[4]->insert(true);
+        else
+            res_columns[4]->insert(false);
+
+
+        /// isro command
+        /// The server will respond with "ro" if in read-only mode or "rw" if not in read-only mode.
+        String isro_output = sendFourLetterCommand(host, port, "isro");
+        if (isro_output == "ro")
+            res_columns[5]->insert(true);
+        else
+            res_columns[5]->insert(false);
+
+        /// mntr command
+        String mntr_output = sendFourLetterCommand(host, port, "mntr");
+        std::map <String, String> mntr_responses_map = getTokens(mntr_output, '\t');
+
+        /// /* 6 */{"version", std::make_shared<DataTypeString>(), "The ZooKeeper version."},
+        res_columns[6]->insert(mntr_responses_map["zk_version"]);
+
+        // /* 7 */ {"avg_latency", std::make_shared<DataTypeUInt64>(), "The average latency."},
+        res_columns[7]->insert(stoi(mntr_responses_map["zk_avg_latency"]));
+
+        // /* 8 */ {"max_latency", std::make_shared<DataTypeUInt64>(), "The max latency."},
+        res_columns[8]->insert(stoi(mntr_responses_map["zk_max_latency"]));
+
+        //  /* 9 */ {"min_latency", std::make_shared<DataTypeUInt64>(), "The min latency."},
+        res_columns[9]->insert(stoi(mntr_responses_map["zk_min_latency"]));
+
+        // /* 10 */ {"packets_received", std::make_shared<DataTypeUInt64>(), "The number of packets received."},
+        res_columns[10]->insert(stoi(mntr_responses_map["zk_packets_received"]));
+
+        // /* 11 */ {"packets_sent", std::make_shared<DataTypeUInt64>(), "The number of packets sent."},
+        res_columns[11]->insert(stoi(mntr_responses_map["zk_packets_sent"]));
+
+        // /* 12 */ {"outstanding_requests", std::make_shared<DataTypeUInt64>(), "The number of outstanding requests."},
+        res_columns[12]->insert(stoi(mntr_responses_map["zk_outstanding_requests"]));
+
+        // /* 13 */ {"server_state", std::make_shared<DataTypeString>(), "Server state."},
+        res_columns[13]->insert(mntr_responses_map["zk_server_state"]);
+
+        auto followers_in_string = mntr_responses_map["zk_followers"];
+        int followers = 0;
+        if (!followers_in_string.empty())
+        {
+            followers = stoi(followers_in_string);
+        }
+
+        ///* 14 */ {"is_leader", std::make_shared<DataTypeUInt8>(), "Is this zookeeper leader."},
+        if (followers)
+            res_columns[14]->insert(true);
+        else
+            res_columns[14]->insert(false);
+
+        ///* 15 */ {"znode_count", std::make_shared<DataTypeUInt64>(), "The znode count."},
+        res_columns[15]->insert(stoi(mntr_responses_map["zk_znode_count"]));
```

Comment:
we have very fast and nice `parse<int>` function.

Reply:
addressed it